wiki/Guides/Adding_Custom_Hit_Sounds/id.md

```diff
@@ -0,0 +1,45 @@
+Menambahkan Kustom Hit sound
```

Comment:
`Menambahkan Hit Sound Kustom` (Indonesian grammar uses `(noun) (adjective)` as opposed to English's `(adjective) (noun)` in the original article's title.)